packages/app-bridge/src/registries/commands/NewPublication.ts

```diff
@@ -0,0 +1,11 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import type { DispatchHandlerParameter } from '../../AppBridge';
+import type { CommandRegistry } from './CommandRegistry';
+
+export const openNewPublication = (
+    options?: CommandRegistry['openNewPublication'],
```

Comment:
```suggestion
    payload: CommandRegistry['openNewPublication'],
```
Same here can it be `undefined`, also renamed `options` to `payload` to align with line 10

Reply:
Fixed 👌